go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/jaypipes/ghw v0.13.0
 	github.com/k8stopologyawareschedwg/noderesourcetopology-api v0.1.2
 	github.com/k8stopologyawareschedwg/podfingerprint v0.2.2
-	github.com/klauspost/cpuid/v2 v2.2.8
+	github.com/klauspost/cpuid/v2 v2.2.9-0.20241026200132-ee7cba987ff4
```

Comment:
Would it be better to wait for a `v2.2.9` release that includes this feature? Perhaps it would be worth reaching out to the cpuid maintainer to request a new release.

Reply:
OTOH it would be nice to get this into NFD v0.17

Reply:
alright, then let's take this https://github.com/kubernetes-sigs/node-feature-discovery/pull/1949 in first and rebase this one afterwards

I just asked the `cpuid` maintainer for a new release and he was kind enough to do it right now.

Reply:
@ozhuraki Can you rebase your branch, it is now merged?